WalletWasabi/Crypto/ZeroKnowledge/LinearRelation/Equation.cs

```diff
@@ -0,0 +1,61 @@
+using NBitcoin.Secp256k1;
+using System.Linq;
+using WalletWasabi.Crypto.Groups;
+using WalletWasabi.Helpers;
+
+namespace WalletWasabi.Crypto.ZeroKnowledge.LinearRelation
+{
+	// Each proof of a linear relation consists of multiple knowledge of
+	// representation equations, all sharing a single witness comprised of several
+	// secrets.
```

Comment:
Feel free to ignore: I don't understand the math behind all this but this comment does not help me really to understand it better. I would expect something like: `This class represent a single equation <something> where P is public point, G_1, G_2, ..., G_n are generator points of secp256k1 elliptic curve.` (Made-up comment just to make my point clear.)

Reply:
i think this makes a bit more sense in the context of #4280, where the linear relation proof system is completed, and also before that humourless robocop codefactor made me move the property declaration after the constructor and the comment on line 24 came right after

i will try to improve the clarity in that PR